### PR TITLE
Issue#3150 - Esc and click outside is working to close searchbox

### PR DESCRIPTION
**Pull request by seAfnan**
## Expected behavior
twenty-bug-solved

### CODE REVIEW

**Comment by Kanav-Arora** [comment]
Hi @seAfnan 
I think this 'esc to cancel' can be written as just 'esc' in a greyish text. @Bonapara what do you think?

File: packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx [hunk 1]
```diff
@@ -109,6 +121,15 @@ export const CommandMenu = () => {
     [toggleCommandMenu, setSearch],
   );
 
+  useScopedHotkeys(
+    'esc,Esc,Escape,escape',
```
Comment (martmull) [inline packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx:125]: Think you just need `'esc'` here. With `'esc,Esc,Escape,escape'`, the `closeCommandMenu` is called 4 times

File: packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx [hunk 2]
```diff
@@ -109,6 +121,15 @@ export const CommandMenu = () => {
...
+    () => {
+      closeCommandMenu();
+    },
+    AppHotkeyScope.CommandMenuOpen,
```
Comment (martmull) [inline packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx:129]: Not sure you need to add another hotKeyScope for this feature. Why not using `AppHotkeyScope.CommandMenu`?

**Review by martmull — CHANGES_REQUESTED** [review]
Hey @seAfnan, thank you for your contribution. Great job! Think we need to validate the design with @Bonapara. I have also left some comments

File: packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx (hunk 2 above)
Comment (martmull) [reply packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx:129]: Ok, I have my answer for this one -> https://github.com/twentyhq/twenty/issues/3150#issuecomment-1869066747
Resolving the comment

File: packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx [hunk 3]
```diff
@@ -61,6 +62,16 @@ export const StyledInput = styled.input`
   }
 `;
 
+const StyledCancelText = styled.span`
+  color: gray;
+  font-size: 12px;
```
[the remark this answers is not in the document]
Comment (seAfnan) [reply packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx:67]: That's true, I will change it.

File: packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx [hunk 4]
```diff
@@ -61,6 +62,16 @@ export const StyledInput = styled.input`
...
+  margin-right: 8px;
+  margin-top: 4px;
```
[the remark this answers is not in the document]
Comment (seAfnan) [reply packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx:69]: Yes you are right, there should be a handsome margin and I will do that.

File: packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx (hunk 1 above)
Comment (seAfnan) [reply packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx:125]: Ok, will try that too.

**Comment by seAfnan** [comment]
Above changes done.

**Comment by martmull** [comment]
Thank you for your changes, looks good to me

**Review by martmull — APPROVED** [review]

**Pull request merged**